src/fswAlgorithms/effectorInterfaces/solarArrayReference/solarArrayReference.c

```diff
@@ -50,7 +50,15 @@ void SelfInit_solarArrayReference(solarArrayReferenceConfig *configData, int64_t
 */
 void Reset_solarArrayReference(solarArrayReferenceConfig *configData, uint64_t callTime, int64_t moduleID)
 {
-
+    if (!NavAttMsg_C_isLinked(&configData->attNavInMsg)) {
+        _bskLog(configData->bskLogger, BSK_ERROR, "Error: solarArrayReference.attNavInMsg wasn't connected.");
```

Comment:
:large_orange_diamond: "Error: " is redundant. The bsk logger prepends the string BSK_ERROR to the message.

Reply:
Fixed it.